tests/paramsServer.test.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable max-len */
+'use strict';
```

Comment:
you could disable strict the same way

Reply:
I don't see any benefits in disabling `use strict` rule for files, or you didn't mean it?